src/routes/Upgrade/UpgradeViewContainer.js

```diff
@@ -36,7 +36,7 @@ import UpgradeView from './UpgradeView'
 function validatePrivateKey(address: ?Address) {
   return ({ privateKey }: UpgradePrivateKeyFormFieldValues): UpgradePrivateKeyFormFieldErrors => {
     if (!address) {
-      throw new WalletInconsistentDataError('Address is empty', '')
+      throw new WalletInconsistentDataError(null, 'Address is empty')
```

Comment:
`throw new WalletInconsistentDataError('Address is empty')`
why `null` in first param?